src/serac/physics/state/state_manager.hpp

```diff
@@ -58,33 +58,33 @@ class StateManager {
   static FiniteElementDual newDual(FiniteElementVector::Options&& options  = {},
                                    const std::string&             mesh_tag = default_mesh_name_);
 
-  /**
-   * @brief Factory method for creating a new QuadratureData object
-   * @tparam T The type of the per-qpt data
-   * @param[in] name The name of the quadrature data field
-   * @param[in] p The order of the quadrature rule
-   * @param[in] mesh_tag A string that uniquely identifies the mesh on which the quadrature data is to be defined
-   * @see QuadratureData::QuadratureData
-   */
-  template <typename T>
-  static QuadratureData<T>& newQuadratureData(const std::string& name, const int p,
-                                              const std::string& mesh_tag = default_mesh_name_)
-  {
-    auto& datacoll = datacolls_.at(mesh_tag);
-    if (is_restart_) {
-      auto field = datacoll.GetQField(name);
-      syncable_data_.push_back(std::make_unique<QuadratureData<T>>(*field));
-      return static_cast<QuadratureData<T>&>(*syncable_data_.back());
-    } else {
-      SLIC_ERROR_ROOT_IF(datacoll.HasQField(name),
-                         axom::fmt::format("Serac's datacollection was already given a qfield named '{0}'", name));
-      syncable_data_.push_back(std::make_unique<QuadratureData<T>>(mesh(mesh_tag), p, false));
-      // The static_cast is safe here because we "know" what we just inserted into the vector
-      auto& qdata = static_cast<QuadratureData<T>&>(*syncable_data_.back());
-      datacoll.RegisterQField(name, &(qdata.QFunc()));
-      return qdata;
-    }
-  }
+  ///**
+  // * @brief Factory method for creating a new QuadratureData object
+  // * @tparam T The type of the per-qpt data
+  // * @param[in] name The name of the quadrature data field
+  // * @param[in] p The order of the quadrature rule
+  // * @param[in] mesh_tag A string that uniquely identifies the mesh on which the quadrature data is to be defined
+  // * @see QuadratureData::QuadratureData
+  // */
+  // template <typename T>
```

Comment:
I wasn't sure what to do with this code, it wasn't compiling because I removed the old quadrature data logging. However, that capability didn't appear to be tested, or allow us to do restarts anyway.

Reply:
If it's not working or testing, I say we delete it. We can always get it from the git history.